Index: mysite/dixi/admin.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.contrib import admin\r\n\r\n\r\nfrom .models import *\r\n\r\nadmin.site.register(Product)\r\nadmin.site.register(Material)\r\nadmin.site.register(Price)\r\nadmin.site.register(Category)\r\nadmin.site.register(Gallery)\r\nadmin.site.register(Reviews)\r\nadmin.site.register(Cart)\r\nadmin.site.register(Order)\r\nadmin.site.register(CartProduct)\r\n\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mysite/dixi/admin.py b/mysite/dixi/admin.py
--- a/mysite/dixi/admin.py	(revision 95a5173a192a28b0a1fb1f3191350268cdfdd8ca)
+++ b/mysite/dixi/admin.py	(date 1641218884963)
@@ -12,6 +12,6 @@
 admin.site.register(Cart)
 admin.site.register(Order)
 admin.site.register(CartProduct)
-
+admin.site.register(Size)
 
 
